test/unit/three_scale/swagger/specification_test.rb

```diff
@@ -24,6 +24,21 @@ class ThreeScale::Swagger::SpecificationTest < ActiveSupport::TestCase
     assert "", specification.base_path
   end
 
+  test 'petstore oas 3.0' do
+    doc = load_fixture 'pet-store-3.0'
+    specification = ThreeScale::Swagger::Specification.new(doc)
+
+    assert specification.valid?, 'specification should be valid'
+  end
+
+  test 'oas 3.0 base path' do
```

Comment:
I think that `base_path` does not make anymore sense for OAS 3.0
Validation should happen for all servers, BUT we do not use it yet in our application. We still can validate all servers and picking arbitrarily the first server as base_path

Reply:
https://github.com/3scale/porta/pull/1532#discussion_r369671937